engines/sci/detection_tables.h

```diff
@@ -1133,7 +1133,7 @@ static const struct ADGameDescription SciGameDescriptions[] = {
 		Common::EN_ANY, Common::kPlatformMacintosh, ADGF_MACRESFORK, GUIO4(GUIO_NOSPEECH, GAMEOPTION_PREFER_DIGITAL_SFX, GAMEOPTION_ORIGINAL_SAVELOAD, GAMEOPTION_FB01_MIDI)	},
 
 #ifdef ENABLE_SCI32
-#define GUIO_HOYLE5_DEMO GUIO3(GUIO_NOSPEECH, \
+#define GUIO_HOYLE5 GUIO3(GUIO_NOSPEECH, \
                                GUIO_NOASPECT, \
                                GAMEOPTION_ORIGINAL_SAVELOAD)
```

Comment:
I just talked with @OmerMor 
I also talked with @wjp a bit about maybe checking in case GAMEOPTION_ORIGINAL_SAVELOAD is not active, and then skipping ::save/::restore patching. It seems patching atm doesn't cause issues and it seems that it's never called, so we can keep that in for now.

GAMEOPTION_ORIGINAL_SAVELOAD should be removed, GUIO_NOLAUNCHLOAD should be added (as already mentioned).
Other than that it looks fine.